kong/db/schema/entities/routes.lua

```diff
@@ -122,6 +125,23 @@ else
                         then_match = { len_eq = 0 },
                         then_err = "'snis' can only be set when 'protocols' is 'grpcs', 'https', 'tls' or 'tls_passthrough'",
                       }},
-                    },
+      { custom_entity_check = {
+        field_sources = { "path_handling" },
+        fn = function(entity)
+          if entity.path_handling == "v1" then
+            if kong_router_flavor == "traditional" then
+              deprecation("path_handling='v1' is deprecated and will be removed in future version, " ..
```

Comment:
Shall we add a deprecation after version and a removal version?
https://github.com/Kong/kong/blob/e2b048845429694f009b73d3a482cd0182af494c/kong/conf_loader/init.lua#L681-L685